src/main/java/frc/team5115/Classes/Software/AutoAimAndRange.java

```diff
@@ -0,0 +1,132 @@
+package frc.team5115.Classes.Software; 
+
+import frc.team5115.Constants;
+import frc.team5115.Classes.Hardware.*;
+import frc.team5115.Classes.Software.PhotonVision;
+import org.photonvision.PhotonPoseEstimator;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.estimator.PoseEstimator;
+import edu.wpi.first.math.estimator.SwerveDrivePoseEstimator;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team5115.Classes.Software.Drivetrain;
+
+import frc.team5115.Constants.*;
+import java.util.Optional;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+
+
+
+public class AutoAimAndRange extends SubsystemBase{
+    HardwareDrivetrain j;
```

Comment:
If these variables are only used inside this class, they should be `private`. Not including the scope here makes them `package` scope, which means that they can be called from any class in `frc.team5115.Classes.Software`, which I'm not sure is intentional.